app/controllers/facilities_controller.rb

```diff
@@ -113,6 +114,14 @@ def transactions_with_search
     paginate_order_details
   end
 
+  # GET /facilities/:facility_id/disputed_orders
+  def disputed_orders_with_search
```

Comment:
One question I have is do we care about having this new page or could we just have two options in the dropdown, one for reservations and one for orders?